_topic_maps/_topic_map.yml

```diff
@@ -989,6 +989,8 @@ Topics:
     File: cert-manager-operator-install
   - Name: Managing certificates with an ACME issuer
     File: cert-manager-operator-issuer-acme
+  - Name: Enabling monitoring for cert-manager
```

Comment:
@swghosh Would it be okay to use "cert-manager Operator for Red Hat OpenShift" instead of just cert-manager throughout the PR? AFAIK we are dealing with the **operand** here rather than the Operator.

Reply:
Yeah, that was my concern. Since we're actually exposing the operand metrics instead of the operator metrics (which we do not have as of today atleast) I was more aligned towards using operand naming.
However, I'd leave it to @bergerhoffer to decide, what's the best approach here.

Reply:
Yeah it's a tough one because when we initially did the docs for it, they were pretty adamant about making it clear that we're talking about our RH cert-manager operator (and not the upstream cert-manager one). 

I'd recommend using the operator name throughout for the most part, since that's what's done now. And then you can always evaluate if it's important to really get specific about what is the operator vs operand (and then make that change throughout all of the cert-manager doc files).

Reply:
Ack. Updated the PR with "cert-manager Operator for Red Hat OpenShift". Thank you :)